File: Amigo.Tenant.Application.DTOs/Requests/Expense/ExpenseRegisterRequest.cs
using Amigo.Tenant.Application.DTOs.Responses.MasterData;
using System;
using System.Collections.Generic;

namespace Amigo.Tenant.Application.DTOs.Requests.Expense
{
    public class ExpenseRegisterRequest : IEntity
    {
        public int? ExpenseId { get; set; }
        public DateTime? ExpenseDate { get; set; }
        public int? PaymentTypeId { get; set; }
        public int? HouseId { get; set; }
        public int? PeriodId { get; set; }
        public string ReferenceNo { get; set; }
        public string Remark { get; set; }
        public decimal? SubTotalAmount { get; set; }
        public decimal? Tax { get; set; }
        public decimal? TotalAmount { get; set; }
        public bool? RowStatus { get; set; }
        public int? CreatedBy { get; set; }
        public DateTime? CreationDate { get; set; }
        public int? UpdatedBy { get; set; }
        public DateTime? UpdatedDate { get; set; }




    }
}

File: Amigo.Tenant.UI.Web/src/app/amigotenant/expense/dto/expense-register-request.ts
export class ExpenseRegisterRequest
{
        public ExpenseId : number;
        public ExpenseDate : Date;
        public PaymentTypeId : number;
        public HouseId : number;
        public PeriodId : number;
        public ReferenceNo : string;
        public Remark : string;
        public SubTotalAmount : number;
        public Tax : number;
        public TotalAmount : number;
}